src/main/java/frc/robot/FieldConstants.java

```diff
@@ -0,0 +1,194 @@
+        new Translation2d(Units.inchesToMeters(345.428), Units.inchesToMeters(242.855));
+    public static final Translation2d CLOSE_CAGE =
+        new Translation2d(Units.inchesToMeters(345.428), Units.inchesToMeters(199.947));
+
+    // Measured from floor to bottom of cage
+    public static final double DEEP_HEIGHT = Units.inchesToMeters(3.125);
+    public static final double SHALLOW_HEIGHT = Units.inchesToMeters(30.125);
+  }
+
+  public static class CoralStation {
+    public static final Pose2d LEFT_CENTER_FACE =
+        new Pose2d(
+            Units.inchesToMeters(33.526),
+            Units.inchesToMeters(291.176),
+            Rotation2d.fromDegrees(90 - 144.011));
+    public static final Pose2d RIGHT_CENTER_FACE =
+        new Pose2d(
+            Units.inchesToMeters(33.526),
+            Units.inchesToMeters(25.824),
+            Rotation2d.fromDegrees(144.011 - 90));
+  }
+
+  public static class Offsets {
+    public static final Pose3d CAMERA_OFFSET = Pose3d.kZero;
+    public static final double ADJUST_X = Units.inchesToMeters(0);
+    public static final double ADJUST_Y = Units.inchesToMeters(0);
+  }
+
+  public static class Reef {
+    public static final Translation2d CENTER =
+        new Translation2d(Units.inchesToMeters(176.746), Units.inchesToMeters(158.501));
+    public static final double FACE_TO_ZONE_LINE =
+        Units.inchesToMeters(12); // Side of the reef to the inside of the reef zone line
+
+    public static final ArrayList<Pose2d> CENTER_FACES =
+        new ArrayList<>(
+            Arrays.asList(
+                new Pose2d[] {
+                  new Pose2d(
+                      Units.inchesToMeters(144.003),
+                      Units.inchesToMeters(158.500),
+                      Rotation2d.fromDegrees(180)),
+                  new Pose2d(
+                      Units.inchesToMeters(160.373),
+                      Units.inchesToMeters(186.857),
+                      Rotation2d.fromDegrees(120)),
+                  new Pose2d(
+                      Units.inchesToMeters(193.116),
+                      Units.inchesToMeters(186.858),
+                      Rotation2d.fromDegrees(60)),
+                  new Pose2d(
+                      Units.inchesToMeters(209.489),
+                      Units.inchesToMeters(158.502),
+                      Rotation2d.fromDegrees(0)),
+                  new Pose2d(
+                      Units.inchesToMeters(193.118),
+                      Units.inchesToMeters(130.145),
+                      Rotation2d.fromDegrees(-60)),
+                  new Pose2d(
+                      Units.inchesToMeters(160.375),
+                      Units.inchesToMeters(130.144),
+                      Rotation2d.fromDegrees(-120))
+                })); // Starting facing the driver station in clockwise order
+
+    public static final ArrayList<Integer> CENTER_FACES_IDS =
+        new ArrayList<>(Arrays.asList(new Integer[] {7, 6, 11, 10, 9, 8}));
```

Comment:
```suggestion
    public static final List<Integer> CENTER_FACES_IDS = List.of(7, 6, 11, 10, 9, 8);
```
This is more concise.